mergify_engine/tests/functional/test_simulator.py

```diff
@@ -78,6 +76,57 @@ def test_simulator_with_token(self):
             f"#### Rule: assign (assign)\n- [X] `base={self.master_branch_name}`\n\n<hr />"
         )
 
+        mergify_yaml = """pull_request_rules:
+  - name: remove label conflict
+    conditions:
+      - -conflict
+    actions:
+      label:
+        remove:
+          - conflict:
+"""
+
+        r = self.app.post(
+            "/simulator/",
+            json={"pull_request": None, "mergify.yml": mergify_yaml},
+            headers={
+                "Authorization": f"token {config.EXTERNAL_USER_PERSONAL_TOKEN}",
+                "Content-type": "application/json",
+            },
+        )
+        assert r.status_code == 400, r.json()
+        assert r.json() == {
+            "errors_list": [
+                "expected str @ pull_request_rules → 0 → actions → label → remove → 0",
+            ]
+        }
+
+        mergify_yaml = """pull_request_rules:
+  - name: remove label conflict
+    conditions:
+      - -conflict:
+    actions:
+      label:
+        remove:
+          - conflict:
+"""
+
+        r = self.app.post(
+            "/simulator/",
+            json={"pull_request": None, "mergify.yml": mergify_yaml},
+            headers={
+                "Authorization": f"token {config.EXTERNAL_USER_PERSONAL_TOKEN}",
+                "Content-type": "application/json",
+            },
+        )
+        assert r.status_code == 400, r.json()
+        assert r.json() == {
+            "errors_list": [
+                "expected str @ pull_request_rules → 0 → actions → label → remove → 0",
```

Comment:
unrelated but i'm wondering if we could replace 0 by `item 0` to make it more understandable

Reply:
It seems a good idea, I will do it in another PR.